Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -80,9 +83,18 @@ private IEnumerable<KBucket> NonEmptyBuckets
 
         public IEnumerable<BoundPeer> PeersToBroadcast(Address? except)
         {
-            return NonEmptyBuckets
+            var peers = NonEmptyBuckets
                 .Select(bucket => bucket.GetRandomPeer(except))
-                .Where(peer => !(peer is null));
+                .Where(peer => !(peer is null)).ToList();
+            var count = peers.Count;
+            if (count < _minPeersToBroadcast)
+            {
+                peers.AddRange(Peers
+                    .Where(peer => except is null || !peer.Address.Equals(except.Value))
```

Comment:
Can we rewrite as below?

```suggestion
                    .Where(peer => !except?.Equals(peer.Address)
```

Reply:
It's `System.Nullable<bool>` so can't be used in `where` phrase.